### PR TITLE
Enhance attester logging

### PR DESCRIPTION
**Pull request by terencechain**
As we have seen in attester logging, it gets unbearable when multiple validator keys registered within a single client. So many duplicated `Submitted new attestation` logs when the difference is just the public key

Screen Shot 2019-12-31 at 1 13 14 PM

This PR updates attester logging experience to be more sane. As it waits till every attester has submitted unaggregated attestation or until end of context. Then it logs `Submitted new unaggregated attestation` with new key field `ValidatorIndices`

Screen Shot 2019-12-31 at 1 17 56 PM

Another pro from this is gives us more space to log a few more key fields such as committee index, source and target check points. This greatly enhances debugability

### CODE REVIEW

**Comment by mcdee** [comment]
With this change it seems like it would be possible for node A to generate an attestation and broadcast it, and node B to receive the attestation before node A has logged its creation.  This could confuse people attempting to trace movement of attestations.  Perhaps retain a log message at the point the attestation is created, but at debug (or even trace) level?

**Comment by terencechain** [comment]
> With this change it seems like it would be possible for node A to generate an attestation and broadcast it, and node B to receive the attestation before node A has logged its creation. This could confuse people attempting to trace movement of attestations. Perhaps retain a log message at the point the attestation is created, but at debug (or even trace) level?

Sounds good to me

File: validator/client/runner.go [hunk 1]
```diff
@@ -115,6 +116,7 @@ func run(ctx context.Context, v Validator) {
 				wg.Wait()
 				span.End()
 			}()
+			v.LogAttestationsSubmitted()
```
Comment (mcdee) [inline validator/client/runner.go]: Shouldn't this be after the `span.End()` above?

That said, looking at the code I don't see where the goroutines call `wg.Done()` so not sure that goroutine ever gets past `wg.Wait()`.

Comment (terencechain) [reply validator/client/runner.go]: It doesn't get logged if it's:
```go
go func() {
	wg.Wait()
	v.LogAttestationsSubmitted()
	span.End()
}()
```

Any suggestions?

Comment (mcdee) [reply validator/client/runner.go]: Try adding `wg.Done()` at line 111 and see if that helps.

Comment (terencechain) [reply validator/client/runner.go]: Thanks! 👍

**Pull request merged**